foundry/canvas.d.ts

```diff
@@ -81,57 +109,53 @@ declare class Canvas {
    */
   protected _pendingOperationNames: Set<string>;
 
-  /* -------------------------------------------- */
-
   /**
    * Create the layers of the game Canvas.
    * @param stage - The primary canvas stage
    */
   protected _createLayers(stage: PIXI.Container): void;
 
-  /* -------------------------------------------- */
-  /*  Properties and Attributes
-  /* -------------------------------------------- */
-
   /**
    * A mapping of named CanvasLayers.
    * This mapping is defined in the order that layers must be drawn.
    */
-  static get layers(): Record<string, CanvasLayer>;
-
-  /* -------------------------------------------- */
+  static get layers(): {
+    background: any; // TODO: BackgroundLayer
+    tiles: any; // TODO: TilesLayer
+    drawings: any; // TODO: DrawingsLayer
+    grid: GridLayer;
+    walls: any; // TODO: WallsLayer
+    templates: any; // TODO: TemplateLayer
+    notes: any; // TODO: NotesLayer
+    tokens: TokenLayer;
+    lighting: LightingLayer;
+    sounds: any; // TODO: SoundsLayer
+    sight: SightLayer;
+    effects: any; // TODO: EffectsLayer
+    controls: any; // TODO: ControlsLayer
+  } & Partial<Record<string, CanvasLayer>>;
```

Comment:
These should be either `ConstructorOf` or `typeof` since it returns the constructor, rather than an instance.

Reply:
you're right, I'll adjust it.

Reply:
done